### DISCUSSION TITLE
Meaning of content’s anchors

### DISCUSSION THREAD

**Discussion by Leedehai** [cetz-package/cetz#440] [Q&A] [answered]
Hi - I'm kinda confused about how `content()`'s parameter `anchor` works. For instance, in the manual, to put the content _above_ a given line the anchor value is set to `"south"`. But.. it doesn't mean the `south` of the line, right? (the line's coordinate being `("line.start", 0.5, "line.end")` is also a bit confusing, btw, but I think it's using the "interpolation" form of coordinate)

Also, I have seen cases where if users would like to label a circle, they need to write `content(a, $ ... $, anchor: "bottom")` to put the label _above_ the circle. This also surprises me, since I'd expect the anchor to be `"top"`.

It would help a lot if there's an explanation in the manual. Thanks!😊

**Comment by johannes-wolf** [comment, marked as answer]
The anchor parameter of elements is the origin of the element. For content, the default origin is `"center"`, but it gets translated by `default-anchor - anchor` if `anchor:` is set.

Yes, the `(a, ratio, b)` form is the short interpolation coordinate form. Long would be `(a: a, b: b, number: ratio, abs: bool?, angle: angle?)`, see the documentation.

#441 and #442.
